tests/mock.data.js

```diff
@@ -31,3 +31,10 @@ export const createRequestBody = ({issuer, vc = validVc}) => {
 export function createISOTimeStamp(timeMs = Date.now()) {
   return new Date(timeMs).toISOString().replace(/\.\d+Z$/, 'Z');
 }
+
+export const invalidIssuerTypes = new Map([
+  ['null', null],
+  ['a boolean', true],
+  ['a number', 4],
+  ['an empty Array', []]
```

Comment:
This makes it seem as if these normative statements exist in the specification, and they do not. More here: https://github.com/w3c-ccg/vc-api-issuer-test-suite/pull/34/files#r1014750792

Reply:
Addressed here: https://github.com/w3c-ccg/vc-api-issuer-test-suite/pull/35/commits/d4fd287bcd9b817bca9410bbd9262ae32edbc2a2